CONTRIBUTORS.md

```diff
@@ -18,3 +18,4 @@
 - [Thomas Windt](https://github.com/WOLFI3654)
 - [Tim Solokha](https://github.com/TimSolokha)
 - [viewless](https://github.com/viewless)
+- [Aditya Narayan Shahi](https://github.com/Shahi9935)
```

Comment:
Hey Aditya, the file clearly mentions `Please follow ALPHABETICAL ORDER`. Please add your name accordingly.